alpha/apps/kaltura/lib/microservice/MicroServiceChecklist.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Checklist for Micro Service
+ */
+
+class MicroServiceChecklist extends MicroServiceBaseService
+{
+	public static $host = 'checklist';
+	public function __construct()
+	{
+		$this->hostName = self::$host;
+		parent::__construct();
+	}
+
+	public static function buildServiceUrl($hostName, $serviceName , $isApi = true)
+	{
+		$url = parent::buildServiceUrl($hostName, false, false);
```

Comment:
Better to pass serviceName as null, since buildServiceUrl expects the second param to be string